app/src/main/res/layout/edit_field.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/editTaskField"
+    android:layout_margin="@dimen/fab_margin"
+    android:visibility="invisible"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/editTextField"
+        android:layout_width="261dp"
+        android:layout_height="68dp"
```

Comment:
Fixed dp heights and widths are never good. Try to do this without fixing the height and width.

Reply:
Try using Rations or Percentages, or constrain properly with 0dp width